.storybook/addons.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import '@storybook/addon-knobs/register';
```

Comment:
How come we don't have to do this for other addons? Namely `@storybook/addon-actions` or `@storybook/addon-links`....

Reply:
That's a good point, I've blindly followed the instructions 😅 let me check!

Reply:
that file is for the configurations of the different tabs in the sidebar if there is more and one addon, we need to create the addons file
![screen shot 2018-05-31 at 2 24 47 pm](https://user-images.githubusercontent.com/3579758/40784835-a9cce6fc-64de-11e8-9dc3-448a815fe7d6.png)